components/restapi/core/component.go

```diff
@@ -85,31 +87,29 @@ const (
 	// GET returns the output IDs of all UTXO changes.
 	RouteCommitmentByIndexUTXOChanges = "/commitments/by-index/:" + restapipkg.ParameterSlotIndex + "/utxo-changes"
 
-	// RouteAccountsByAcciuntID is the route for getting an account by its accountID.
-	// GET returns the account details.
-	RouteAccountsByAcciuntID = "/accounts/:" + restapipkg.ParameterAccountID
+	// RouteBlockIssuanceCredits is the route for getting block issuance credits balance for an account.
+	// GET returns the block issuance credits balance.
+	RouteBlockIssuanceCredits = "/accounts/:" + restapipkg.ParameterAccountID
 
-	// RouteAccountMana is the route for getting an account mana by its accountID.
-	// GET returns the account mana details.
-	RouteAccountMana = "/accounts/:" + restapipkg.ParameterAccountID + "/mana"
+	// RouteCongestion is the route for getting the current congestion state and all account related useful details as block issuance credits.
+	// GET returns the congestion state raleted to the specified account.
+	RouteCongestion = "/accounts/:" + restapipkg.ParameterAccountID + "/congestion"
 
-	// RoutePeer is the route for getting peers by their peerID.
-	// GET returns the peer
-	// DELETE deletes the peer.
-	RoutePeer = "/peers/:" + restapipkg.ParameterPeerID
+	//RouteStaking is the route for getting informations about the current stakers.
+	// GET returns the stakers.
+	RouteStaking = "/staking"
 
-	// RoutePeers is the route for getting all peers of the node.
-	// GET returns a list of all peers.
-	// POST adds a new peer.
-	RoutePeers = "/peers"
+	// RouteStakingAccount is the route for getting an account by its accountID.
+	// GET returns the account details.
+	RouteStakingAccount = "/staking/:" + restapipkg.ParameterAccountID
 
-	// RouteControlDatabasePrune is the control route to manually prune the database.
-	// POST prunes the database.
-	RouteControlDatabasePrune = "/control/database/prune"
+	// RouteRewards is the route for getting the rewards for staking of delegation based on sstaking account or delegaion output.
+	// GET returns the rewards.
+	RouteRewards = "/rewards/:" + restapipkg.ParameterAccountID
```

Comment:
why is this an accountID and not an outputID?

Reply:
that's mistake, we use later outputID, will correct that